amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -2544,14 +2559,23 @@ type Expenditure @model {
   nativeFundingPotId: Int!
     @index(
       name: "byNativeFundingPotId"
-      queryField: "getExpendituresByNativeFundingPotId"
+      queryField: "getExpendituresByNativeFundingPotIdAndColony"
+      sortKeyFields: ["colonyId"]
```

Comment:
This is a bug I missed the first time. Since funding pot ids aren't unique across colonies we should also query the colony id whenever querying this (currently only happens in one place in block ingestor)